cypress/integration/lists_test/enrichment_test.js

```diff
@@ -0,0 +1,99 @@
+Cypress.on('uncaught:exception', (err, runnable) => {
+    if (err.message.includes('Can not create edge')) {
+        // Cytoscape.js tools are throwing this - ignore for now.
+        return false
+    }
+})
+
+describe("Enrichment Test", function(){
+    beforeEach(function(){
+        cy.visit("/");
+        cy.createListFromFile("all_genes.csv","All genes"); //This file is downloaded from querybuilder.
+        cy.createListFromFile("enrichment_gene.csv","Enrichment analysis genes");
+        cy.get(".enrichment").should("exist").within(() => {
+            cy.get(".dropdown").click();
+            cy.get(".list-selection").eq(0).click();
+        })
+    });
+
+    it("can view and download all enrichment results", function(){
+        cy.get(".enrichment").should("exist").within(() => {
+            cy.get('.enrichment-header').within(() => { 
+                cy.get('[type="checkbox"]').check();
+                cy.wait(500);
+                cy.get('[type="checkbox"]').should("be.checked"); 
+            })
+            cy.intercept("POST","/biotestmine/service/query/results").as("tableLoad");
+            cy.contains("View").click();
+            cy.wait("@tableLoad");
+        })
+        cy.get('.pagination-label').should("include.text","172 rows");
+        cy.get(".enrichment-category").eq(0).within(() => { 
+            cy.get(".icon-download").click();
+        })
+        cy.readFile("cypress/downloads/Enrichment Results Gene Ontology Enrichment.tsv").then(newResult => {
+            cy.readFile("cypress/fixtures/all_enrichment_result.tsv").should("eq",newResult);
+        })
```

Comment:
In this and the other "view and download" tests, can you switch the order so it downloads first, and then *views*?